aries-site/src/data/wcag/tip.json

```diff
@@ -0,0 +1,512 @@
+[
+  {
+    "rule": "1.1.1",
+    "label": "Non-text content",
+    "Column1": "conditional",
+    "notes": "Implementor needs to apply the correct aria role"
+  },
+  {
+    "rule": "1.2.1",
+    "label": "Audio-only and Video-only (Prerecorded)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.2",
+    "label": "Captions (Prerecorded)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.3",
+    "label": "Audio Description or Media Alternative (Prerecorded)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.4",
+    "label": "Captions (Live)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.5",
+    "label": "Audio Description (Prerecorded)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.6",
+    "label": "Sign Language (Prerecorded)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.7",
+    "label": "Extended Audio Description (Prerecorded)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.8",
+    "label": "Media Alternative (Prerecorded)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.9",
+    "label": "Audio-only (Live)",
+    "Column1": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.1",
+    "label": "Info and Relationships",
+    "Column1": "conditional",
+    "notes": "Tip needs to be given an aria-label, aria-labelledby, or aria-describedby so that the tip can clearly be associated with its trigger."
```

Comment:
Can we file issues for the following:

- on grommet-site add an issue to improve the documentation to make it more clear that an `id` needs to be passed through `dropProps` and the tooltip trigger should refence this id via `aria-describedby`

- on grommet create an issue to check all components that use Tip internally and make sure we follow the `id` + `aria-describedby` pattern.

- on grommet create an issue to update storybook examples to demonstrate the `id` + `aria-describedby` pattern.